scripts/update_xcode_config_cpp_checks.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+prefixConfigurationFlags="OTHER_CFLAGS=$(OTHER_CFLAGS) -"
```

Comment:
Do you mind if I touch up this python code? It's not following a lot of python standards.

Reply:
Feel free to do so. I'm very new to python though

Reply:
Also feel free to merge [your PR](https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/814) excluding the configuration changes. The script here should help to add those configurations in future